ui/src/app/pages/analysis/notebook-list.tsx

```diff
@@ -36,23 +50,68 @@ export const NotebookList = withCurrentWorkspace()(class extends React.Component
     notebookList: FileDetail[],
     notebookNameList: string[],
     creating: boolean,
-    loading: boolean
+    loading: boolean,
+    delay: number,
+    cloneLoadingNotebookMsg: boolean;
   }> {
+  private interval: NodeJS.Timeout;
   constructor(props) {
     super(props);
-    this.state = {notebookList: [], notebookNameList: [], creating: false, loading: false};
+    this.state = {
+      notebookList: [],
+      notebookNameList: [],
+      creating: false,
+      loading: false,
+      delay: 3000,
+      cloneLoadingNotebookMsg: false
+    };
   }
 
   componentDidMount() {
     this.props.hideSpinner();
     profileApi().updatePageVisits({page: 'notebook'});
-    this.loadNotebooks();
+    this.confirmNotebookTransferIsDone();
   }
 
-  componentDidUpdate(prevProps) {
+  componentDidUpdate(prevProps, prevState) {
     if (this.workspaceChanged(prevProps)) {
       this.loadNotebooks();
     }
+    if (prevState.delay !== this.state.delay) {
```

Comment:
could you explain what this check does?

Reply:
We do not need this actually now since i am not changing the delay